tests/0056-balanced_group_mt.c

```diff
@@ -178,9 +178,9 @@ static void rebalance_cb (rd_kafka_t *rk,
 
 
 static void get_assignment (rd_kafka_t *rk_c) {
+        if(!rk_c)
```

Comment:
Why this check? From what I see rk_c is never passed as NULL. (also: space-before-paren..)